src/stylesheets/components/navbar.scss

```diff
@@ -33,8 +33,8 @@
 }
 
 .navbar__icon__group {
-	height: 70px;
-	width: 70px;
+	height: 55px;
```

Comment:
Might be better to make all these sizes in terms of vh. If you do the same values for both widthhs and heights, then everything will scale based on device height. It's more of a long-term solution than setting an absolute value

Reply:
Or even % values could work